.azure/bicep/config/web-app-settings.bicep

```diff
@@ -0,0 +1,45 @@
+param keyVaultName string
+param appInsightsName string
+param appName string
+param appSettings object = {}
+
+var secretRef = '@Microsoft.KeyVault(VaultName=${keyVaultName};SecretName='
+
+var sharedSettings = {
+  OutboundEmail__FromName: 'hutch'
+  OutboundEmail__Provider: 'sendgrid'
+  OutboundEmail__SendGridApiKey: '${secretRef}sendgrid-api-key)'
+  OutboundEmail__FromAddress: 'noreply@hutch.com'
+  // Environment for deployment (Development or Production)
+  ASPNETCORE_ENVIRONMENT: '${secretRef}ASPNETCORE_ENVIRONMENT)'
+  // Paths to SSL certificates
+  ASPNETCORE_Kestrel__Certificates__Default__Path: '${secretRef}ASPNETCORE_Kestrel__Certificates__Default__Path)'
+  ASPNETCORE_Kestrel__Certificates__Default__KeyPath: '${secretRef}ASPNETCORE_Kestrel__Certificates__Default__KeyPath)'
+  // Connection string to database
+  ASPNETCORE_ConnectionStrings__Default: '${secretRef}ASPNETCORE_ConnectionStrings__Default)'
```

Comment:
- Don't need the `ASPNETCORE_` prefix here.
- can secret keys use `kebab-case` for consistency?

also remember that the keyvault is shared across the apps, so clearer key names might be needed for settings that are manager or agent specific

Reply:
I can make variables kebab case. The ones that start `ASPNETCORE` I think need the prefix because they're pulled out of the environment rather than from the `AppSettings.*.json` files. https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-7.0#default-host-configuration-sources